Libraries/LibWeb/SVG/SVGScriptElement.cpp

```diff
@@ -40,15 +43,75 @@ void SVGScriptElement::process_the_script_element()
     if (m_already_processed || !in_a_document_tree())
         return;
 
-    auto inline_script = child_text_content();
+    IGNORE_USE_IN_ESCAPING_LAMBDA String script_content;
+    auto script_url = m_document->url();
 
-    // FIXME: 2. If the 'script' element references external script content, then the external script content
-    //           using the current value of the 'xlink:href' attribute is fetched. Further processing of the
-    //           'script' element is dependent on the external script content, and will block here until the
-    //           resource has been fetched or is determined to be an invalid IRI reference.
+    // 2. If the 'script' element references external script content, then the external script content
+    //    using the current value of the 'xlink:href' attribute is fetched. Further processing of the
+    //    'script' element is dependent on the external script content, and will block here until the
+    //    resource has been fetched or is determined to be an invalid IRI reference.
     if (has_attribute(SVG::AttributeNames::href) || has_attribute_ns(Namespace::XLink.to_string(), SVG::AttributeNames::href)) {
-        dbgln("FIXME: Unsupported external fetch of SVGScriptElement!");
-        return;
+        auto href_value = href()->base_val();
+
+        script_url = document().parse_url(href_value);
+        if (!script_url.is_valid()) {
+            dbgln("Invalid script URL: {}", href_value);
+            return;
+        }
+
+        auto& vm = realm().vm();
+        auto request = Fetch::Infrastructure::Request::create(vm);
+        request->set_url(script_url);
+        request->set_destination(Fetch::Infrastructure::Request::Destination::Script);
+        request->set_mode(Fetch::Infrastructure::Request::Mode::NoCORS);
```

Comment:
> When fetching external resources from the Internet, user agents must use a [potentially CORS-enabled request](https://html.spec.whatwg.org/multipage/urls-and-fetching.html#create-a-potential-cors-request) as defined in HTML [[HTML](https://www.w3.org/TR/SVG2/refs.html#ref-html)] with the corsAttributeState as follows:
>    For an ‘[href](https://svgwg.org/specs/animations/#HrefAttribute)’ reference on an ‘[image](https://www.w3.org/TR/SVG2/embedded.html#ImageElement)’ element or ‘[script](https://www.w3.org/TR/SVG2/interact.html#ScriptElement)’ element, the CORS state specified by the ‘[crossorigin](https://www.w3.org/TR/SVG2/embedded.html#ImageElementCrossoriginAttribute)’ attribute.
    
It's fine for now to use NoCORS here, at least until Luke's PR is merged. A FIXME saying to read the `crossorigin` attribute would be appropriate.

Reply:
Added a FIXME